crates/pallet-evm-swap/Cargo.toml

```
@@ -0,0 +1,66 @@
+[package]
+name = "pallet-evm-swap"
```

```suggestion
name = "pallet-native-to-evm-swap"
```

I think it makes more sense if we'll be splitting the crates.

---

Got it. Should we call precompile logic as `precompile-evm-to-native-swap` ? Or, just `precompile-evm-swap` is enough ?

---

`precompile-evm-to-native-swap` makes sense for symmetry

---

Done